doc/nrf/libraries/others/dm.rst

```diff
@@ -0,0 +1,70 @@
+.. _mod_dm:
+
+Distance Measurement
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Distance Measurement module provides an integration of :ref:`nrfxlib:nrf_dm` into |NCS|.
+
+Overview
+********
+
+The Distance Measurement module is independent of the communication protocol used.
+The :ref:`nrfxlib:mpsl` is used to access the radio module which is required by the :ref:`nrfxlib:nrf_dm` library.
+During the allocated tiemslot, a distance measurement is performed.
+
+Using Distance Measurement
+**************************
+
+To use Distance Measurement module, enable it using the :kconfig:`CONFIG_NRF_DM` Kconfig option and initialize it in your :file:`main.c` file.
+After synchronizing the nodes between which the measurement is to be performed, add the measurement request.
+When the measurement data are available, the callback function will be called.
```

Comment:
```suggestion
The callback function is called when the measurement data is available.
```
You could be more specific. Which callback function?

Reply:
Fixed.